### ISSUE TITLE
Menopause present

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10091]
**Preferred term label:**
Menopause present

**Synonyms**
Perimenopausal

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Mammary duct ectasia

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000059
- label: Perimenopausal
- parents: [Past medical history (HP:0032443)](https://hpo.jax.org/app/browse/term/HP:0032443)
- pmid: PMID:29952797
- def: Perimenopause, or the menopausal transition, encompasses that period of time during which physiologic changes mark progression toward a woman's final menstrual period. This phase begins with the onset of menstrual irregularities and continues until a woman reaches menopause, or one year after amenorrhea has occurred. Perimenopause can last for a variable amount of time, the median of which is four years.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10091
- synonyms: Menopause present(NONE)

**Comment by pnrobinson** [comment]
@MickeySegal 
HPO Id corrected to HP:6000059

**Comment by MickeySegal** [comment]
I thought this was corrected to HP:0040406

**Comment by pnrobinson** [comment]
@MickeySegal 
sorry, you are right, the correct term id is HP:0040406
